app/views/content/research.md

```diff
@@ -122,6 +130,13 @@ keywords:
   - Research
 ---
 
+## Career Countdown: exploring how students and recent graduates are feeling about taking their next steps
+
+Kindred Agency commissioned research company Censuswide to carry out a survey on behalf of the Get Into Teaching campaign exploring the views of final year university students and recent graduates (from the last three years) in 
```

Comment:
@EllieNodder Richard provided a slightly shorter summary paragraph on the ticket - can we use this instead?

Kindred Agency commissioned research company Censuswide to carry out a survey on behalf of Get Into Teaching exploring the views of final year university students and recent graduates. The purpose was to gain insights into how this cohort feel when it comes to making decisions about their future career and what factors are influencing them.

Reply:
oops yeah, missed that, will update now